src/models/chan.js

```diff
@@ -136,11 +136,29 @@ Chan.prototype.removeUser = function(user) {
 	this.users.delete(user.nick.toLowerCase());
 };
 
-Chan.prototype.toJSON = function() {
-	var clone = _.clone(this);
-	clone.users = []; // Do not send user list, the client will explicitly request it when needed
-	clone.messages = clone.messages.slice(-100);
-	return clone;
+Chan.prototype.getFilteredClone = function(lastActiveChannel, lastMessage) {
```

Comment:
Could you add a comment to describe what those arguments `lastActiveChannel` and `lastMessage` are to the function? Just looking at the signature, I'm not seeing what they would do.

Reply:
Done.